src/main/webapp/js/feed-loader.js

```diff
@@ -0,0 +1,57 @@
+
+
+/** Fetches messages and add them to the page. */
+  function fetchMessages(){
+    const url = '/feed';
+    fetch(url).then((response) => {
+      return response.json();
+    }).then((messages) => {
+      const messageContainer = document.getElementById('message-container');
+      if(messages.length == 0){
+       messageContainer.innerHTML = '<p>There are no posts yet.</p>';
+      }
+      else{
+       messageContainer.innerHTML = '';  
+      }
+      messages.forEach((message) => {  

```

Comment:
Can we move this into the else statement? It wont run for message length 0 as is, but its a bit easier to read inside the else.

Reply:
I made this change